docs/topics/mta-web-configuring-and-running-an-application-analysis.adoc

```diff
@@ -64,8 +64,8 @@ You can analyze more than one application at a time against more than one transf
 ====
 Attaching custom rules is optional if you have already attached a migration target to the analysis. If you have not attached any migration target, you must attach rules.
 ====
-** In *Manual* mode, type the name of the required rule or search for it by clicking *Add Rules*. Then either drag and drop the relevant files or select them from their directory.
-** In *Repository* mode, you can add rule files from Git. To do that, you have to know the files' exact names and paths.
+** In *Manual* mode, click *Add Rules*. Then either drag and drop the relevant files or select them from their directory.
```

Comment:
@sbeskin-redhat could you please add: "Click Add"?
Thanks
![Selection_543](https://user-images.githubusercontent.com/60384172/234790346-025376c1-614a-40a2-b631-9ab18f672b38.png)

Reply:
@mguetta1  Added